workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-batch-object-definition/batch/object-definition.batch-engine-data.json

```diff
@@ -149,6 +150,7 @@
 				{
 					"DBType": "String",
 					"businessType": "Picklist",
+					"defaultValue": "eMEA",
```

Comment:
Shouldn't this be "EMEA"?

Reply:
Hey @ethib137 , if you try to create a picklist entry, the first letter of any key must be in lowercase.